app/models/navigation/BreadCrumbs.scala

```diff
@@ -15,7 +15,7 @@ object BreadCrumbs {
   sealed trait BreadCrumb
   final case class BCDynamicText(cn: String => String) extends BreadCrumb
   final case class BCDynamicNamedLink(cn: String => String, cl : String => Call) extends BreadCrumb
-  final case class BCDynamicMultiNamedLink(cn: String => String, cl : (String,String) => Call) extends BreadCrumb
+  final case class BCDynamicMultiNamedLink(cn: String => String, cl : (String, List[String]) => Call) extends BreadCrumb
```

Comment:
I change second parameter of dynamic multi named link from string to list, such we can pass more parameters.